### PR TITLE
HW20_WebTableTests_Playwright

### PR DESCRIPTION
**Pull request by Svitlytsya**

### CODE REVIEW

File: PlaywrightDemoQA/WebTableTests.cs [hunk 1]
```diff
@@ -15,12 +15,72 @@ public async Task DeleteRow()
...
+        var context = await browser.NewContextAsync();
+
+        var page = await context.NewPageAsync();
+        await page.GotoAsync("https://demoqa.com/webtables");
+
+        await page.Locator(".rt-tr-group")
+            .Filter(new() { Has = page.GetByRole(AriaRole.Gridcell, new() { Name = "Alden", Exact = true }) })
+            .GetByTitle("Edit").ClickAsync();
+        await page.GetByPlaceholder("Last Name").FillAsync("Canter");
+        await page.GetByPlaceholder("Salary").FillAsync("13200");
```
Comment (vldemyan) [inline PlaywrightDemoQA/WebTableTests.cs]: Task "increase by 10%" means that you need to read current value and increase it by 10%. Test should work for different current values but not for single current value

File: PlaywrightDemoQA/WebTableTests.cs [hunk 2]
```diff
@@ -15,12 +15,72 @@ public async Task DeleteRow()
...
+        await page.GetByPlaceholder("Last Name").FillAsync("Sangu");
+        await page.GetByPlaceholder("name@example.com").FillAsync("elchinsangu@ask.com");
+        await page.GetByPlaceholder("Age").FillAsync("42");
+        await page.GetByPlaceholder("Salary").FillAsync("35000");
+        await page.GetByPlaceholder("Department").FillAsync("Finance");
+        await page.GetByRole(AriaRole.Button, new() { Name = "Submit" }).ClickAsync();
+
+        var newRow = page.Locator(".rt-tr-group")
+            .Filter(new() { HasText = "Elchin" })
+            .Filter(new() { HasText = "Sangu" })
```
Comment (vldemyan) [inline PlaywrightDemoQA/WebTableTests.cs]: "Filter" is needed only once, to find new row. After that you need to verify other values in this row, like 
await Assertions.Expect(newRow).ToContainTextAsync(lastName);
await Assertions.Expect(newRow).ToContainTextAsync(email);
etc.

File: PlaywrightDemoQA/WebTableTests.cs (hunk 1 above)
Comment (Svitlytsya) [reply PlaywrightDemoQA/WebTableTests.cs]: Fixed, Thanks.

File: PlaywrightDemoQA/WebTableTests.cs (hunk 2 above)
Comment (Svitlytsya) [reply PlaywrightDemoQA/WebTableTests.cs]: Fixed, Thanks.

File: PlaywrightDemoQA/WebTableTests.cs [hunk 3]
```diff
@@ -15,12 +15,79 @@ public async Task DeleteRow()
...
+
+        await page.Locator(".rt-tr-group")
+            .Filter(new() { Has = page.GetByRole(AriaRole.Gridcell, new() { Name = "Alden", Exact = true }) })
+            .GetByTitle("Edit").ClickAsync();
+        await page.GetByPlaceholder("Last Name").FillAsync("Canter");
+        await page.GetByPlaceholder("Salary").FillAsync(newSalary.ToString());
+        await page.GetByRole(AriaRole.Button, new() { Name = "Submit" }).ClickAsync();
+
+        await Assertions.Expect(page.GetByRole(AriaRole.Grid)).ToContainTextAsync("Canter");
+        await Assertions.Expect(page.GetByRole(AriaRole.Grid)).ToContainTextAsync("13200");
```
Comment (vldemyan) [inline PlaywrightDemoQA/WebTableTests.cs]: this should be .ToContainTextAsync(newSalary.ToString());

**Review by vldemyan — DISMISSED** [review]

File: PlaywrightDemoQA/WebTableTests.cs (hunk 3 above)
Comment (Svitlytsya) [reply PlaywrightDemoQA/WebTableTests.cs]: ok, thank you, fixed.

**Review by vldemyan — APPROVED** [review]

**Pull request merged**